docs/config/README.md

```diff
@@ -949,7 +1315,7 @@ The `line_break` module separates the prompt into two lines.
 
 ### Options
 
-| Variable   | Default | Description                                                        |
+| Option     | Default | Description                                                        |
 | ---------- | ------- | ------------------------------------------------------------------ |
 | `disabled` | `false` | Disables the `line_break` module, making the prompt a single line. |
 
```

Comment:
Should be remove as it's handled by the prompt format.

Reply:
Nevermind.
The line_break module still exist but should by removed as it's basically useless.

Reply:
I'll open a new PR to remove the module.

Reply:
I think that almost nobody is going to use `$all` because if we still have the line break in there it will mess up peoples custom prompts. For example:
```toml
# Use custom format
format = """
[┌───────────────────>](bold green)
[│](bold green)$directory$rust$package
[└─>](bold green) """
```
Replacing the `$directory$rust$package` with `$all` would completely mess up the prompt and while you can hide something like the `character` module by clearing it's format you can't easily just hide the `line_break`.

Or a second option would be just removing `$all` outright and just setting the default to something like this (based on the current default):
```toml
format = """

$username\
$hostname\
$kubernetes\
$directory\
$git_branch\
$git_commit\
$git_state\
$git_status\
$hg_branch\
$docker_context\
$package\
$dotnet\
$elixir\
$elm\
$erlang\
$golang\
$haskell\
$java\
$julia\
$nim\
$nodejs\
$ocaml\
$php\
$purescript\
$python\
$ruby\
$rust\
$terraform\
$zig\
$nix_shell\
$conda\
$memory_usage\
$aws\
$env_var\
$crystal\
$cmd_duration\
$custom\
n\
$jobs\
$battery\
$time\
$character
"""
```

Then people could easily copy this remove the `\n` and build the custom format around that.

Reply:
At the moment the default prompt is `\n$all` and the way to disable the prefix new line is to change the default prompt to `$all` since we are removing the `add_newline` configuration option. If we remove `$all` I think we should add a nice way to disable the prefixed new line. One of the nice things about the current setup is that if the user is happy with the default prompt order then when we release new modules then they "just work" and the user doesn't need to make any changes to their configuration.

Reply:
> > Replacing the $directory$rust$package with $all would completely mess up the prompt and while you can hide something like the character module by clearing it's format you can't easily just hide the line_break.
> 
> I think the following should still work
> 
> ```toml
> [line_break]
> disabled = true
> ```

I actually forgot we had that. In that case we should just leave the `line_break` module.